addons/project/static/src/views/project_task_form/project_task_form_controller.js

```diff
@@ -44,10 +44,14 @@ export class ProjectTaskFormController extends FormController {
             return;
         }
 
+        const recordId = this.props.resId || this.model.config.resId;
+        if (!recordId) {
+             return;
+        }
```

Comment:
```suggestion
        const recordId = this.props.model.record.resId
```

Reply:
this.props.resId === The first open record in the form view. So when you do next page it never changes.

this.props.model.record.resId === The record  currently displayed in the form view

Reply:
Wow, such a helpful knowledge,  i didn't know this

Reply:
> Wow, such a helpful knowledge, i don't know this

Reply:
Generally, adding a timeout to a test is an error. issue:
- Slows down the test suite (3sec isn't much but on thousands of runbots it's expensive)
- Adds indeterminism. The test could fail indeterminately.

Solution:
- Check that an interface element has changed to validate that your save is done ?

Reply:
@FrancoisGe Hello, your code doesn't work. The ` this.props.model` is undefined, you can test it by yourselves. Can i keep mine ?

Reply:
@duong77476-viindoo Sorry it s "this.model"

Reply:
@FrancoisGe yes and it should be `this.model.root.resId`, other places use that